README.md

```diff
@@ -19,6 +24,78 @@ This will prompt for your password because it is
 using HTTPS on port 443 which is a privileged port.
 
 To change the owner (ex. root) of a table:
+```bash
 psql -d launchpad
 alter table {table-name} owner to {new-owner}
 press ctrl-d
+```
+
+## Environment Variables
+
+The following environment variables are used for the database and the API:
+
+```bash
+POSTGRES_USER=
+POSTGRES_PASSWORD=
+POSTGRES_HOST=db
+POSTGRES_DB=launchpad
+POSTGRES_PORT=5432
```

Comment:
The defaults are set in the .env file, so you don't have to set any of these save user and password.